### PR TITLE
Cleanup/pulsing coax cleanup

### PR DESCRIPTION
**Pull request by vertiq-jordan**
This is the first of a few PRs that are going to come your ways. This first one is an update of the files involved in the pulsing coax project. Most of these files are shared in the pulsing copter project. The next PR will be adding the pulsing copter files.

This PR is mostly cosmetic changes. I've added comments where needed, and I've run the provided auto-format script on all of our added files. There are some non-cosmetic changes made based on my discussion with Luca.

1. Removed the motor direction bitmask group entry
2. Deleted the ardupilot client and all includes of it
3. Deleted commented code (sort of cosmetic but sort of not)

I also completed the steps and ensured that I am able to spin a motor with pulsing through the motor test. The motor spins, pulses, and coasts as commanded.

### CODE REVIEW

**Review by vertiq-luca — APPROVED** [review]
Everything looks good for the non-formatting changes. Assuming it compiles then the formatting changes are good to go too!

File: libraries/AP_IQ_Motor/AP_IQ_Motor.cpp [hunk 1]
```diff
@@ -25,13 +31,6 @@ const AP_Param::GroupInfo AP_IQ_Motor::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("TELEM", 2, AP_IQ_Motor, _telemetry_bitmask, 0),
 
-    // @Param: MOT_DIR
-    // @DisplayName: Motor Direction Bitmask
-    // @Description: This sets the direction of each motor. 0 is CCW, 1 is CW.
-    // @Range: 0 65535
-    // @User: Advanced
-    AP_GROUPINFO("DIR", 3, AP_IQ_Motor, _motor_dir_bitmask, 0),
```
Comment (iq-fred) [inline libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:33]: I'm a bit out of the loop, so I may have some obvious questions. Is this just a vestigial thing that has no purpose anymore?

File: libraries/AP_IQ_Motor/AP_IQ_Motor.cpp [hunk 2]
```diff
@@ -80,8 +76,8 @@ void AP_IQ_Motor::init(void)
 
 uint8_t AP_IQ_Motor::add_client(ClientAbstract *new_client)
 {
-    if (_client_size < _client_limit) // TODO check if this client type already exists and don't add if it does. Instead return the pointer?
-    {
+    //Make sure that there is enough space to add a new client
+    if (_client_size < _client_limit) { // TODO check if this client type already exists and don't add if it does. Instead return the pointer?
```
Comment (iq-fred) [inline libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:80]: Is doing this TODO something that is going to happen in one of your upcoming PRs?

File: libraries/AP_IQ_Motor/AP_IQ_Motor.cpp [hunk 3]
```diff
@@ -105,34 +100,32 @@ uint8_t AP_IQ_Motor::find_client(ClientAbstract **set_client, uint8_t client_typ
...
 
-    if (iq_uart == nullptr)
-    {
+    //If we are ever unable to find an instance of our communication protocol, stop trying to update
+    if (iq_uart == nullptr) {
         return;
     }
 
     // TODO: generate general broadcast output message to esc prop from channels array
     // TODO: send telemetry request for current telemetry motor
```
Comment (iq-fred) [inline libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:115]: Same thing with these TODOs, are they planned to happen in PRs after this cosmetic one?

**Review by iq-fred — APPROVED** [review]

File: libraries/AP_IQ_Motor/AP_IQ_Motor.cpp (hunk 1 above)
Comment (vertiq-jordan) [reply libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:33]: Yep, the motor direction bitmask doesn't actually get used anywhere in our control through ardupilot

File: libraries/AP_IQ_Motor/AP_IQ_Motor.cpp (hunk 2 above)
Comment (vertiq-jordan) [reply libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:80]: I was not planning on working on any of the TODOs in this sprint, but want to keep them around for future development. I think trying to further development would be feature creep beyond the defined task of "clean-up and test existing IFCI ardupilot code."

Comment (iq-fred) [reply libraries/AP_IQ_Motor/AP_IQ_Motor.cpp:80]: Ok, thats fine with me. Just need to watch it out for if the Ardupilot people question the TODOs, not sure if they usually do or don't.

**Pull request merged**